RELEASE_NOTES.rst

```diff
@@ -388,7 +388,7 @@ Here are the important changes and additions:
 - Settings unification - most bears have seen API changes. Settings' names
   are now consistent across bears. This supports backwards
   compatibility, however (but with a deprecation notice). You can find the
-  whole list `here <http://dpaste.com/3EP5GCV>`__.
+  whole list `here <https://pastebin.com/1QQ8gN1p>`__.
```

Comment:
This will also disappear in the future.  We need to include the content in our own repo somewhere.

Good news is that your paste brings back memories -- yes, this is what was being talked about.

Now we just need to plan how to include it in the repo so it isnt lost again.

Reply:
If we want to include this in the repository we could:

- Include the list as a collapsible in RELEASE_NOTES.rst like in the description of this PR
- Save this list as an issue on this repository
- Save the content as a gist (not recommended, gists probably disappear with accounts)

The content could be represented as a table to make it more readable in markdown.

Reply:
Just occurred to me, this file is an rst, GitHub won't display any collapsible content in rst files. Only markdown. So without it being collapsible, we'd end up with this huge list/table as is in the file.

Still a go?